Content.Shared/SubFloor/SubFloorHideSystem.cs

```diff
@@ -46,6 +49,7 @@ public override void Initialize()
             SubscribeLocalEvent<SubFloorHideComponent, ComponentShutdown>(OnSubFloorTerminating);
             SubscribeLocalEvent<SubFloorHideComponent, AnchorStateChangedEvent>(HandleAnchorChanged);
             SubscribeLocalEvent<SubFloorHideComponent, ComponentHandleState>(HandleComponentState);
+            SubscribeLocalEvent<SubFloorHideComponent, InteractUsingEvent>(OnInteractionAttempt);
```

Comment:
There's no guarantee that this subscription will be run before any other interactions, so it's not guaranteed to block them. It also won't stop empty-handed interactions.

Doing this properly probably requires changing  `ActionBlockerSystem.CanInteract` & the `InteractionAttemptEvent` event, which is probably best left for another PR. So what ypu're doing here is probably good enough for now.